src/pykeen/datasets/analysis.py

```diff
@@ -301,8 +304,9 @@ def iter_unary_patterns(
         rev_ht = {(t, h) for h, t in ht}
         confidence = len(ht.intersection(rev_ht)) / support
         yield PatternMatch(r, "symmetry", support, confidence)
-        confidence = len(ht.difference(rev_ht)) / support
-        yield PatternMatch(r, "anti-symmetry", support, 1 - confidence)
+        # confidence = len(ht.difference(rev_ht)) / support
+        confidence = 1 - confidence
```

Comment:
why do you redefine confidence? Why not just write it in the next line as 1 - confidence